### ISSUE TITLE
3.0.1 release reports "3.0.1rc5" via ompi_info

### ISSUE THREAD

**Issue by amckinstry** [open-mpi/ompi#5082]
Downloaded the 3.0.1 release from https://www.open-mpi.org/software/ompi/v3.0/

However in the VERSION file, "greek=rc5".

**Comment by bwbarrett** [comment]
Can you share the URL you downloaded Open MPI 3.0.1 from and the sha1sum of the file?  When I download either the .tar.gz and .tar.bz2 files, I don't see rc5 in the VERSION file.

**Comment by jsquyres** [comment]
Ditto for what @bwbarrett said -- I see no greek version in the official 3.0.1 tarballs from https://www.open-mpi.org/software/ompi/v3.0/.

**Comment by amckinstry** [comment]
Huh, I confirm that the version now on the website is correct. 
I don't have the tarball I downloaded, so I can't verify now.  A local error looks likely.

My local workflow is to download the upstream tarball and commit to a local git repo as branch 'upstream', so my logs show I downloaded the tarball on April 3, but I see no sign of rc5 on the webpage (and 3.0.1 was published March 29.)

So feel free to close this bug unless you think something might have happened on your end.
